### PR TITLE
Making the ListView Scrollable to the Right *and* to the Left

### PR DESCRIPTION
**Pull request by psygo**
Apparently, you can scrolll to the left by simply replacing `ListView` by `InfiniteListView` (Issue #10), which comes from another package called [`infinite_listview`](https://pub.dev/packages/infinite_listview#-readme-tab-).

I think you will need to add the `infinite_listview` to the `pubspec.yaml` file (PR #13), but I'm not sure &mdash; I'm not experienced with putting packages inside packages.

I would suggest testing if this does fill all of the requirements. Perhaps putting it into another branch might be more appropriate.

### CODE REVIEW

**Comment by hinterlandsupplyco** [comment]
Can this get merged in please?

**Comment by iamvivekkaushik** [comment]
This feature is now added to the library in the latest version.

**Comment by psygo** [comment]
Did you merge this separately?

**Comment by iamvivekkaushik** [comment]
No, I added the functionality myself along with other features.

**Comment by psygo** [comment]
Ok, could you summarize exactly what you did in order to get it done without the [`infinite_listview`](https://pub.dev/packages/infinite_listview#-readme-tab-) package?

**Comment by iamvivekkaushik** [comment]
I just added a start date option, now instead of starting from. selection date timeline can potentially start from any date from the past. I also added a separate option for initial selection. You can read the code if you wanna know more.

**Comment by psygo** [comment]
That doesn't sound like exactly the same as the [`infinite_listview`](https://pub.dev/packages/infinite_listview#-readme-tab-) package, but ok, I guess.
